### PR TITLE
Update FAQ content - add new Maker Question [tutorial]

### PR DESCRIPTION
**Pull request by linderr**
added to faq.handlebars
added to en.json

anything else??

### CODE REVIEW

File: views/faq.handlebars [hunk 1]
```diff
@@ -270,6 +270,29 @@
               "makerspace" items to locate the ones that can use your contribution!
             </p>
           </dd>
+          <dt class="btn-link">
+            <h5 class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners">Who are #findthemasks Makers partners?</h5>
+          </dt>
+          <dd class="collapse">
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-1">
```
Comment (nburt) [inline views/faq.handlebars:277]: you will need to update to `data-i18n="[html]ftm-faq-sites-ftm-maker-partners-answer-1"` when you add the links in.

File: views/faq.handlebars [hunk 2]
```diff
@@ -270,6 +270,29 @@
...
+              for maker groups fabricating PPE, and identifying funding sources for groups producing PPE. They are 
+              also collating resources for makers responding to COVID-19.
+            </p>
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-2">
+              #findthemasks works with Shield Net, a group that matches organizations in need of medical face 
+              shields with manufacturers that produce them. They have amassed a wide network of university makerspaces 
+              and commercial manufacturers who are making face shields based on their open-source design.
+            </p>
+            <p class="i18n" data-i18n="ftm-faq-sites-ftm-maker-partners-answer-3">
+             If you are a makerspace of any size, join their face shield network here.
```
Comment (nburt) [inline views/faq.handlebars:291]: if you want these to all be on their own separate lines you'll need them to be in separate <p> tags

File: views/faq.handlebars (hunk 1 above)
Comment (linderr) [reply views/faq.handlebars:277]: the [html] will trigger the country-specific JS replacement??
That is SO cool

File: views/faq.handlebars (hunk 2 above)
Comment (linderr) [reply views/faq.handlebars:291]: Can I just cheat and use <br> ?

**Comment by linderr** [comment]
@nburt 
Awesome. I was just looking through the code to understand how you guys achieve country-specific linking! My understanding from your explanation is that <**a class="add-donation-site-form"**> has been set up to do that JS replacement automatically??? THAT IS SO COOL!!!!!!!!

2. definitely want to help stay under translation quotas.
Do you mean make all text changes before merging the Smartling dozen xx.json PR?
Sorry, I just did that this time to check my work/convince myself this was correct

**Comment by linderr** [comment]
re: US form link on faq.handlebars
I noticed that and went on the live site and tested a few countries, even though the code on faq.handlebars links directly to the US form via <a                href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform"> button </a> 
--- it's listed correctly in en.json as <a class='add-donation-site-form'>button </a> so I guess that's the JS magic you guys did!! so cool!

File: views/faq.handlebars (hunk 1 above)
Comment (linderr) [reply views/faq.handlebars:277]: Okay, so [html] tells the code that there is HTML here! and then retrieves from somewhere else in the code (no idea yet, probably another file?) which tells it what to do with HTML stuff????????

**Comment by linderr** [comment]
I see. The Smartling translation process does not get re-triggered if I make edits after accepting/merging that PR. Okay, got it!

**Comment by linderr** [comment]
I broke this b/c I want to see how it works. Will fix back tmr morning :) thank you!

also anyone else just feel free to do this page haha. I am just learning - ignore this PR other than for lulz

File: views/faq.handlebars (hunk 2 above)
Comment (nburt) [reply views/faq.handlebars]: `<p>` is better for the translators

**Comment by nburt** [comment]
Gonna close this in favor of https://github.com/findthemasks/findthemasks/pull/808 to get this out the door this am

**Pull request closed without merging**